macros/edr/metadata_collection/get_tables_from_information_schema.sql

```diff
@@ -68,3 +67,28 @@
     )
 
 {% endmacro %}
+
+{% macro postgres__get_tables_from_information_schema(schema_tuple) %}
+    {%- set database_name, schema_name = schema_tuple %}
+
+    (with information_schema_tables as (
+
+        select
+            upper(table_catalog) as database_name,
+            upper(table_schema) as schema_name,
+            upper(table_name) as table_name
+        from information_schema.tables
+            where upper(table_schema) = upper('{{ schema_name }}') and upper(table_catalog) = upper('{{ database_name }}')
+
+    )
+
+    select
+        {{ elementary.full_table_name() }} as full_table_name,
+        upper(database_name || '.' || schema_name) as full_schema_name,
+        database_name,
+        schema_name,
+        table_name
+    from information_schema_tables
+    )
```

Comment:
Why is the query wrapped with parantheses?

Reply:
I think the contributor just followed the others' pattern.
There doesn't seem to be a need in those parentheses.
Removed.